daemon/daemon_windows.go

```diff
@@ -625,7 +620,7 @@ func (daemon *Daemon) setDefaultIsolation() error {
 				daemon.defaultIsolation = containertypes.Isolation("hyperv")
 			}
 			if containertypes.Isolation(val).IsProcess() {
-				if system.IsWindowsClient() && osv.Build < 17763 {
+				if system.IsWindowsClient() && build < osversion.RS5 {
```

Comment:
I would remove `build := osversion.Build()` above and use `osversion.Build()` directly here as it was done in other similar places.

Reply:
Ah, yes, I think I left it because I thought this was run in a loop, but it will only hit this one once; I'll update